bower.json

```diff
@@ -21,19 +21,19 @@
   "homepage": "https://github.com/PolymerElements/gold-cc-cvc-input",
   "ignore": [],
   "dependencies": {
-    "iron-icon": "PolymerElements/iron-icon#2.0-preview",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#2.0-preview",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#2.0-preview",
-    "paper-input": "PolymerElements/paper-input#2.0-preview",
-    "paper-styles": "PolymerElements/paper-styles#2.0-preview",
-    "polymer": "Polymer/polymer#^2.0.0-rc.2"
+    "iron-icon": "PolymerElements/iron-icon#^2.0.0",
```

Comment:
Can you please make the deps hybrid? See [paper-input/bower.json](https://github.com/PolymerElements/paper-input/blob/master/bower.json) versions and resolutions for reference